src/main/steps/respondent/hub-page/template.njk

```diff
@@ -3,7 +3,16 @@
 {% extends "../../applicant1/hub-page/template.njk" %}
 
 {% block soleLatestUpdate %}
-  {% if progressionIndex == 11 %}
+{% if progressionIndex == 15 %}
+    <p class="govuk-body">{{ conditionalOrderPronounced.line1 }}</p>
+    <p class="govuk-inset-text">{{ conditionalOrderPronounced.line2 }}</p>
+    <p class="govuk-body">{{ conditionalOrderPronounced.line3 }}</p>
+    <p class="govuk-body">{{ conditionalOrderPronounced.line4 }}
+      <a class="govuk-link" href={{conditionalOrderPronounced.downloadReference}}
+      download="Certificate-of-Entitlement">
+      {{conditionalOrderPronounced.line5}}</a>
+    </p>
+  {% elif progressionIndex == 11 %}
```

Comment:
i missed this as well but this page will also need to be considered due to the added states.